gestiune_stocuri_magazin/src/product_categories/BooksProduct.java

```diff
@@ -0,0 +1,28 @@
+package product_categories;
+
+import distributors.Distributor;
+
+public class BooksProduct extends Product {
```

Comment:
Book can be more relevant as name - since it inherits Product, can be avoided from the name, also at singular it is best practice to name it